pulpcore/app/migrations/0082_add_manage_roles_permissions.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.8 on 2021-10-21 10:58
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0081_reapplabel_group_permissions'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='task',
+            options={'permissions': [('manage_roles_task', 'Can manage role assignment')]},
```

Comment:
Does it make sense to have have unique descriptions for perms and which will say where I can manage role assignment? on which object?
I'm not sure how obvious it is for a user who looks at the list of available perms.

Reply:
You are suggesting that?
```suggestion
            options={'permissions': [('manage_roles_task', 'Can manage role assignment on task')]},
```
The codename already has the distinction (as it is necessary to distinguish `core.manage_roles_task` from `core.manage_roles_groups`).

Reply:
yes